
 \documentclass[25pt, a0paper, portrait, margin=0mm, innermargin=15mm,
     blockverticalspace=15mm, colspace=15mm, subcolspace=8mm]{tikzposter} %Default values for poster format options.
     
\usepackage{tabularx,booktabs,adjustbox} % For tables
\usepackage{wrapfig,anyfontsize,lipsum}
\linespread{1.2}
\usetikzlibrary{calc,fit,arrows,decorations.pathmorphing,backgrounds,fit,positioning}
\usetikzlibrary{shapes.symbols}

\usepackage{color,colortbl}
\definecolor{Cblack}{rgb}{0,0,0}
\definecolor{Corange}{rgb}{0.9,0.6,0}
\definecolor{Cskyblue}{rgb}{0.35,0.7,0.9}
\definecolor{Cbluegreen}{rgb}{0,0.6,0.5}
\definecolor{Cyellow}{rgb}{0.95,0.9,0.25}
\definecolor{Cblue}{rgb}{0,0.45,0.7}
\definecolor{Cvermillion}{rgb}{0.8,0.4,0}
\definecolor{Cpurple}{rgb}{0.8,0.6,0.7}

\newcommand{\Cblack}[1]{\textcolor{Cblack}{#1}}
\newcommand{\Corange}[1]{\textcolor{Corange}{#1}}
\newcommand{\Cskyblue}[1]{\textcolor{Cskyblue}{#1}}
\newcommand{\Cbluegreen}[1]{\textcolor{Cbluegreen}{#1}}
\newcommand{\Cyellow}[1]{\textcolor{Cyellow}{#1}}
\newcommand{\Cblue}[1]{\textcolor{Cblue}{#1}}
\newcommand{\Cvermillion}[1]{\textcolor{Cvermillion}{#1}}
\newcommand{\Cpurple}[1]{\textcolor{Cpurple}{#1}}

 \newcommand{\magenta}[1]{\textcolor{magenta}{#1}}
 \newcommand{\teal}[1]{\textcolor{teal}{#1}}
  \newcommand{\gray}[1]{\textcolor{gray}{#1}}

% tikz colour settings
\tikzset{pop0/.style={red!50!yellow},pop1/.style={violet!80},pop2/.style={olive!70!green}}

\definecolorpalette{myColorPalette} {
\definecolor{colorOne}{named}{teal}
\definecolor{colorTwo}{named}{white}
\definecolor{colorThree}{named}{cyan}
}

 \tikzposterlatexaffectionproofoff %shows small comment on how the poster was made at bottom of poster

 % Commands
 \newcommand{\bs}{\textbackslash}   % backslash
 \newcommand{\cmd}[1]{{\bf \color{red}#1}}   % highlights command
% \newcommand{\crossmark}{\ding{55}}

 % Title, Author, Institute
 \title{{\bf Efficient simulation of identity-by-descent and ancestry in large datasets}}
 \author{{\bf Georgia Tsambos}}
 \institute{{\bf University of Melbourne, Australia}}

 % -- PREDEFINED THEMES ---------------------- %
 % Choose LAYOUT:  Default, Basic, Rays, Simple, Envelope, Wave, Board, Autumn, Desert,
 \usetheme{Autumn}
\usecolorstyle[colorPalette=myColorPalette]{Denmark}


 \begin{document}

     \maketitle
\block[bodyverticalshift=-2cm]{}{
   {\large\teal{
 Georgia Tsambos (1, 2), Peter Ralph (3), Jerome Kelleher (4), Stephen Leslie (1, 2, 5), Damjan Vukcevic (1, 2).
(1) School of Mathematics and Statistics, University of Melbourne, Australia (2) Melbourne Integrative Genomics, University of Melbourne, Australia, (3) Department of Mathematics, University of Oregon, United States, (4) Big Data Institute, University of Oxford, United Kingdom, (5) School of Biosciences, University of Melbourne, Australia.\\[7mm]
Presenting author: gtsambos (at) student.unimelb.edu.au}
}
}
   
   %%% BLOCK 0  
 \block[titleoffsety=1cm,bodyverticalshift=-20cm]{0. Introduction}{
 }
 \begin{columns}
 \begin{subcolumns}
   \subcolumn{31} \block[bodyoffsetx=1.5cm,bodyverticalshift=-5cm]{}{
{\fontsize{34}{35}\selectfont To assess the performance of methods in population genetics, we often wish to simulate realistic genetic datasets while retaining detailed information about the history of the simulated genomes.
This poster briefly describes how we can efficiently simulate genetic information with full information about the common ancestry of particular genomic segments, as well as information about the populations that these segments have been inherited from.
Although the software is still under development in \texttt{tskit} [1], our progress to date suggests that our methods are scaleable and fast enough to be useful in high-powered studies of subtle demographic questions.}\\[3mm] 
    }
 \end{subcolumns}
 \end{columns}

%%% BLOCK 1
     \begin{columns}%blocks will be placed into columns
         \column{.5}
         \block[roundedcorners=40,titleoffsety=-3cm,bodyoffsety=-3cm,bodyverticalshift=-1cm]{1. The data structure: tree sequences}{
\begin{center}
{\tt\small
\ldots GTAACGCGATAAGA\Cskyblue{T}ATTAGCCCAAAAACACAGACATGG\Cvermillion{A}AATAGCGTA\ldots \\
\ldots GTAACGCGATAAGA\Cvermillion{G}ATTAGCCCAAAAACACAGACATGG\Cskyblue{T}AATAGCGTA\ldots \\
\ldots GTAACGCGATAAGA\Cskyblue{T}ATTAGCCCAAAAACACAGACATGG\Cskyblue{T}AATAGCGTA\ldots \\
\ldots GTAACGCGATAAGA\Cvermillion{G}ATTAGCCCAAAAACACAGACATGG\Cskyblue{T}AATAGCGTA\ldots \\[1mm]
}
\[
\uparrow
\]
\input{pics/tree-sequence.tex}
\[
\Updownarrow
\]
{\small
\renewcommand\baselinestretch{0.8}\selectfont
\begin{tabularx}{.05\textwidth}{p{1.5cm}X}
\toprule
\multicolumn{2}{c}{{\bf Nodes}}\\
\midrule
ID & time  \\
\midrule
0 & 0.0 \\
1 & 0.0\\
2 & 0.0\\
3 & 0.0\\
4 & 1.1\\
5 & 1.8\\
6 & 3.0\\
7 & 3.5\\
8 & 4.0\\
 & \\
 & \\
 \end{tabularx}\quad\quad\begin{tabularx}{.10\textwidth}{p{1.5cm}p{1.5cm}p{2cm}X}
\toprule
\multicolumn{4}{c}{{\bf Edges}}\\
\midrule
left & right & parent & child  \\
\midrule
3.0 & 10.0 & 4 & 2 \\
3.0 & 10.0 & 4 & 3\\
0.0 & 10.0 & 5 & 1\\
0.0 & 3.0 & 5 & 3\\
3.0 & 10.0 & 5 & 4\\
0.0 & 7.0 & 6 & 0\\
0.0 & 7.0 & 6 & 5 \\
7.0 & 10.0 & 7 & 0\\
7.0 & 10.0 & 7 & 5\\
0.0 & 3.0 & 8 & 2\\
0.0 & 3.0 & 8 & 6 \\
\end{tabularx}\quad\quad\begin{tabularx}{.08\textwidth}{p{2.5cm}p{1.5cm}X}
\toprule
\multicolumn{3}{c}{{\bf Mutations}}\\
\midrule
location & time& node  \\
\midrule
1.5 & 2.3 & 5\\
8.9 & 1.2 & 0\\
 & &\\
 & &\\
 & &\\
 & &\\
 & &\\
 & &\\
 & &\\
 & & \\
 & & \\
 \end{tabularx}
}
\end{center}
\vspace{10mm}
The \magenta{tree sequence} data structure [1,2] encodes a complete genealogy for a sample of chromosomes in a succinct set of tables. Compared with traditional sequence-based formats, tree sequences are \emph{compact}, \emph{fast} to process and \emph{informative} of the history of the sample.
%Storing $n = 1\times 10^5$ chromosomes in a compressed VCF requires $\approx$ 50 GB.
%{\fontsize{35}{35}\selectfont However, common haplotypes in a sample are often simply a consequence of some common history. So if we know this history (as we always do in simulations!), storing it directly is often more convenient and efficient than storing the raw haplotypes. This is the key idea behind the \magenta{tree sequence} data structure [1,2], which encodes a complete genealogy for a sample of chromosomes in a succinct set of tables.\\
%Tree sequences offer a few benefits to population geneticists compared with traditional sequence-based file formats:
%
%\begin{itemize}
%\item They can store large simulated datasets extremely compactly.
%
%\item As they hold rich detail about the history of the sample, many important processes can be observed directly from the tree structure.
%
%\item They can be queried and modified extremely quickly.
%\end{itemize}}
     }
     
     %%%% BLOCK 3
     \block[titleoffsety=1cm,bodyoffsety=4cm]{3. Example: total IBD sharing over time}{
     } 
\centering
  \begin{subcolumns}
  \subcolumn{.6}
  \block{}{
  \centering
  \input{pics/demographic-example.tex}
%  \input{pics/new-method/simulation-method-0.tex}
%    \input{pics/new-method/simulation-method-2.tex}
%      \input{pics/new-method/simulation-method-4.tex}
  }
  \subcolumn{.35}
   \block{}{
   \includegraphics[scale=.63]{pics/kinships-10000.png}
   \includegraphics[scale=.63]{pics/kinships-8000.png}
    \includegraphics[scale=.63]{pics/kinships-6000.png}
    \includegraphics[scale=.63]{pics/kinships-4000.png}
\includegraphics[scale=.63]{pics/kinships-2000.png}

  }
  \end{subcolumns}



     \column{.5}
     %%% BLOCK 2
         \block[titleoffsety=-3cm,bodyoffsety=1.5cm]{2. IBD and ancestry in tree sequences}{}  
%          \begin{subcolumns}       
%         \subcolumn{.45}
          \block{}{
          \begin{center}
          \input{pics/ancestry-and-ibd-in-trees.tex}
          \end{center}
{\fontsize{34}{35}\selectfont Questions about inheritance and ancestry can be reframed as questions about the underlying tree sequence that represent our datasets:
          \begin{itemize}
          \item {\it Identity-by-descent: which samples share a common ancestor where?}
          \item {\it Local ancestry: which ancestors have what population labels, and which samples descend from them?}
          \end{itemize}
%          The branch joining a sample node to an ancestral node from a given population shows its ancestry. Any samples with a common ancestor on a tree have inherited the same genomic segment from the ancestor, and thus have an IBD relationship at this location.\\      
          Extracting this information efficiently is challenging due to correlations in genealogical structure between samples, and across chromosomes; 
          in an upcoming paper, we will describe algorithms that allow us to do this.}
          }           
       
\vspace{20cm}

             \block[titleoffsety=0cm,bodyoffsety=0cm]{4. Application: demography inference}{
{\fontsize{34}{35}\selectfont To explore the power of our method, we attempted to recreate some of the important findings from a recent study of Ashkenazi Jewish (AJ) demographic history [3]. This work provided evidence for a recent divergence event between Eastern and Western communities of AJ people. It was estimated that this happened about 15 generations ago.\\[2mm]
Closely following [3], we performed 50 000 simulations of various demographic scenarios. For each simulation, we calculated moments of IBD segment lengths at multiple time points. We used the \texttt{abcrf} package [4] to infer the most plausible demographic scenario, and the \texttt{abc} package [5] with neural-net regression to estimate parameter values.\\

{\bf Results: inference of demographic scenario}
\begin{center}
\begin{tabularx}{.45\textwidth}{p{11cm}p{15cm}X}
\toprule
Posterior probability & Votes for correct scenario & Prior error rate \\
\midrule
$93.36\% \pm 4.69\%$ &	$0.9633 \pm 0.0268$	& $0.0993 \pm 0.0008$\\
\bottomrule\\
\end{tabularx}
\end{center}

{\bf Results: inference of divergence time}
\begin{center}
\begin{tabularx}{.45\textwidth}{p{5cm}p{7cm}p{10cm}X}
\toprule
Mode & True value & Our 95\% HPDI & 95\% HPDI in [3] \\
\midrule
13.59 & 14.93 & $(11.22, 19.90)$ & $(2.00, 29.00)$\\
\bottomrule\\
\end{tabularx}
\end{center}
All simulations and analyses ran on a standard desktop in $<12$ hours.}
}
     \end{columns}

%%% BLOCK 5
 \block[titleoffsety=1cm,bodyoffsety=1cm]{5. Acknowledgements, references and further information}{
 }
  \begin{columns}
 \begin{subcolumns}
    \subcolumn{22} \block[bodyoffsetx=1.5cm,bodyverticalshift=-5cm]{}{\footnotesize
     GT is funded by the Helen Freeman scholarship, the Maurice Belz Fund and the Australian Government's Research Training Scheme.\\[1mm]
     [1] \texttt{https://tskit.readthedocs.io/en/latest/}\\[1mm]
     [2] Kelleher, J., et al. (2016). Efficient Coalescent Simulation and Genealogical Analysis for Large Sample Sizes. PLOS Computational Biology, 12(5).\\[1mm]
     [3] Gladstein, A and Hammer, M. (2019). Substructured Population Growth in the Ashkenazi Jews Inferred with Approximate Bayesian Computation. Molecular Biology and Evolution, 36(6), 1162-1171.\\[1mm] 
    [4] Raynal, L., et al. (2019). ABC random forests for Bayesian parameter inference. Bioinformatics, 35(10), 1720 - 1728.\\[1mm]
	[5] Csillery, K., et al. (2012). Abc: An R package for approximate Bayesian computation (ABC). Methods in Ecology and Evolution, 3(3), 475 - 479.
     }
       \subcolumn{3} \block[bodyoffsetx=-3cm,bodyverticalshift=-5.7cm]{}{
       \begin{center}
       {\small Come say hi!}\\
       \includegraphics[scale=1]{pics/GT_headshot.jpg} 
       \end{center}
       }
       \subcolumn{3} \block[bodyoffsetx=-4cm,bodyverticalshift=-5cm]{}{
       \begin{center}
       \includegraphics[scale=1.2]{pics/unimelb-logo.jpg}
      \end{center}
       }
       \subcolumn{3} \block[bodyoffsetx=-5cm,bodyverticalshift=-6cm]{}{
       \begin{center}
       \includegraphics[scale=1]{pics/tskit-logo.png}
      \end{center}
       }
 \end{subcolumns}
 \end{columns}         

 \end{document}




\endinput
%%
%% End of file `tikzposter-example.tex'.
